src/fiber/graph.rs

```diff
@@ -779,6 +797,22 @@ where
                     continue;
                 }
 
+                // if this is a direct channel, try to load the channel actor state for balance
+                if from == self.source {
+                    if let Some(state) = self
+                        .store
+                        .get_channel_state_by_outpoint(&channel_info.out_point())
+                    {
+                        if amount_to_send > state.to_local_amount {
+                            debug!(
```

Comment:
suggest to remove this debug log, it's too noisy

Reply:
fixed.